.github/workflows/prauto.yml

```
@@ -0,0 +1,67 @@
+name: PR Automation
+on:
+  pull_request:
+    types: [ready_for_review, opened, reopened]
+jobs:
+  pr-auto:
+    runs-on: ubuntu-latest
+    steps:
+      - name: lint pr
+        env:
+          REPO: "cli/cli"
+          NEEDSREVIEWCOL: "MDEzOlByb2plY3RDb2x1bW43MTEwMTI4"
+          PRID: ${{ github.event.pull_request.node_id }}
+          PRBODY: ${{ github.event.pull_request.body }}
+          STAFF: "vilmibm,mislav,samcoe,billygriffin,ampinsk"
+          PRNUM: ${{ github.event.pull_request.number }}
+          PRBASE: ${{ github.event.pull_request.base.ref }}
+          GH_TOKEN: ${{ secrets.AUTOMATION_TOKEN }}
+          PRAUTHOR: ${{ github.event.pull_request.user.login }}
+          # Testing data:
+          #REPO: "vilmibm/testing"
+          #NEEDSREVIEWCOL: "MDEzOlByb2plY3RDb2x1bW4xMTAzNTA3Ng==" # vilmibm/testing "good" column
+        run: |
+          commentPR () {
+            gh -R$REPO pr comment $PRNUM -b "${1}"
+          }
+
+          closePR () {
+            gh -R$REPO pr close $PRNUM
+          }
+
+          addToBoard () {
+            # this exec is an attempt to fix some quote highlighting issues i was having in my editor.
+            bash -c 'gh api graphql -f query="mutation { addProjectCard(input: { projectColumnId: \"$NEEDSREVIEWCOL\", contentId: \"$PRID\" }) { clientMutationId } }"'
+          }
+
+          if echo $STAFF | grep $PRAUTHOR
+          then
+            # TODO this errors if it's already on the board...
+            addToBoard
+            exit 0
+          fi
+
+          if echo "${PRBODY}" | wc -w | grep -E '^[0-9]$'
```

To check if a number is less than 10:
```suggestion
          if [ $(wc -w <<<"$PRBODY") -lt 10 ]
```
I like using `<<<"$var"` instead of `echo "$var"` because the former is safer when `var` contains user data. Echo is unable to print strings like `-e` or `-n` because it will think they are flags. It's a minor edge case and I am probably the only person on the planet who worries about these things :)